src/EventSubscriber/PageHeaderSubscriber.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\localgov_guides\EventSubscriber;
+
+use Drupal\node\Entity\Node;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Drupal\localgov_core\Event\PageHeaderDisplayEvent;
+
+/**
+ * Class PageHeaderSubscriber.
+ *
+ * @package Drupal\localgov_guides\EventSubscriber
+ */
+class PageHeaderSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    return [
+      PageHeaderDisplayEvent::EVENT_NAME => ['setPageHeader', 0],
+    ];
+  }
+
+  /**
+   * Set page title and lede.
+   */
+  public function setPageHeader(PageHeaderDisplayEvent $event) {
+    if ($event->getEntity() instanceof Node &&
+      $event->getEntity()->bundle() == 'localgov_guides_page' &&
+      $event->getEntity()->localgov_guides_parent
+    ) {
+      $overview = $event->getEntity()->localgov_guides_parent->entity;
+      $event->setTitle($overview->getTitle());
+      if ($overview->get('body')->summary) {
+        $event->setLede([
+          '#type' => 'html_tag',
+          '#tag' => 'p',
+          '#value' => $overview->get('body')->summary,
```

Comment:
My twig is horrid, but I assume this gets the correct filtering automagically?

Reply:
This is a render array processed by the core HtmlTag render (https://git.drupalcode.org/project/drupal/-/blob/8.9.x/core/lib/Drupal/Core/Render/Element/HtmlTag.php) so should be safe :wink: